rllib/core/learner/tf/tf_learner.py

```diff
@@ -68,9 +68,7 @@ def __init__(
         super().__init__(framework_hyperparameters=framework_hyperparameters, **kwargs)
 
         self._enable_tf_function = framework_hyperparameters.eager_tracing
-        # the default strategy is a no-op that can be used in the local mode
-        # cpu only case, build will override this if needed.
-        self._strategy = tf.distribute.get_strategy()
+        self._strategy = None
```

Comment:
Can you leave the comment on what `self._strategy` is (or should be when not None)?

Reply:
Strategy is a tf distributed strategy object that is used for the ddp logic.

Reply:
added a param notation.

Reply:
done